src/components/views/rooms/CollapsibleButton.tsx

```diff
@@ -25,18 +25,25 @@ interface ICollapsibleButtonProps extends ComponentProps<typeof MenuItem> {
     title: string;
 }
 
-export const CollapsibleButton = ({ title, className, ...props }: ICollapsibleButtonProps) => {
+export const CollapsibleButton = ({ title, children, className, ...props }: ICollapsibleButtonProps) => {
     const inOverflowMenu = !!useContext(OverflowMenuContext);
     if (inOverflowMenu) {
         return <MenuItem
             {...props}
             className={classNames("mx_CallContextMenu_item", className)}
         >
             { title }
+            { children }
         </MenuItem>;
+    } else {
+        return <AccessibleTooltipButton
+            {...props}
+            title={title}
+            className={className}
+        >
+            { children }
+        </AccessibleTooltipButton>;
     }
```

Comment:
we tend to outdent (and skip else) where the previous branch has a `return`

Reply:
Yuck, ok ;-)

Reply:
Fixed in 6e78236e10